node_pool/main.tf

```diff
@@ -39,6 +39,8 @@ resource "google_container_node_pool" "node_pool" {
     image_type   = "COS"
     disk_size_gb = "${var.disk_size_in_gb}"
     machine_type = "${var.machine_type}"
+    labels       = "${var.node_labels}"
+    disk_type    = "${var.disk_type}"
```

Comment:
To changing either of these settings force terraform to recreate the node_pool? If so they need to be accounted for in the `entropy` `keepers` section. It's an unfortunate byproduct of the `create_before_destroy = true` setting on this resource.

Reply:
I just added keepers for both fo those values.